src/tools/wasm2c.cc

```diff
@@ -79,6 +80,9 @@ static void ParseOptions(int argc, char** argv) {
         s_outfile = argument;
         ConvertBackslashToSlash(&s_outfile);
       });
+  parser.AddOption(
+      'j', "multiple-outputs", "NUM", "Number of output files to write",
```

Comment:
I'm not sure about the use of `j` here.. since since that wasm2c process is not actually running multiple threads, right?    How about `-m` /`--multi-file=N`?

Reply:
Edited in 660bf3daf8b8f33d02e37b74bf947289e920f797. `--num-outputs` is used by both `wasm2c` and `run-spec-wasm2c.py`.